### PR TITLE
IT-5875 Update icons for MediaManager

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Add:
- SIDEBAR_FLIP_SOLID
- FA_SIDEBAR_FLIP
- FA_SQUARE_LIST_SOLID

Remove:
- FA_SIDEBAR

### CODE REVIEW

**Comment by rogatty** [comment]
@fraszczakszymon I see on Figma that we need more icons, both regular and solid versions.
Screenshot 2024-07-23 at 13 29 02

IMO we should wait with it until one of the styling PRs and you could use `FA_SIDEBAR` for now.

**Comment by rogatty** [comment]
And we can remove `FA_SIDEBAR`.

File: lib/js/icons/fontawesome.ts [hunk 1]
```diff
@@ -347,7 +348,7 @@ export const FONTAWESOME_ICONS = {
 	FA_SHARE: faShare,
 	FA_SHARE_NODES: faShareNodes,
 	FA_SHIELD_HALVED: faShieldHalved,
-	FA_SIDEBAR: faSidebar,
+	FA_SIDEBAR_FLIP: faSidebarFlip,
```
Comment (kvas-damian) [inline lib/js/icons/fontawesome.ts:351]: Do we need to import flipped icons? Are we going to import flipped and non flipped version of this asset when we find a use-case for for the latter version? Why we don't use `FA_SIDEBAR` with `flippedHorizontal` prop?

Comment (rogatty) [reply lib/js/icons/fontawesome.ts:351]: `IconButton` doesn't have `flippedHorizontal` prop.

Comment (kvas-damian) [reply lib/js/icons/fontawesome.ts:351]: maybe it's time to expose it

Comment (rogatty) [reply lib/js/icons/fontawesome.ts:351]: I'm cool with that but you should probably talk with UX. 🙂

Comment (kvas-damian) [reply lib/js/icons/fontawesome.ts:351]: https://bethinkteam.slack.com/archives/CFBLF5RGA/p1721897628795609

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**